pkg/cluster/deploystorage.go

```diff
@@ -191,6 +186,144 @@ func (m *manager) deployStorageTemplate(ctx context.Context, installConfig *inst
+					"Microsoft.Network/loadBalancers/probes/join/action",
+
+					"Microsoft.Network/networkInterfaces/write",
+					"Microsoft.Network/networkInterfaces/join/action",
+					"Microsoft.Network/networkInterfaces/delete",
+					"Microsoft.Network/networkInterfaces/ipconfigurations/join/action",
+
+					"Microsoft.Network/publicIPAddresses/write",
+					"Microsoft.Network/publicIPAddresses/delete",
+					"Microsoft.Network/publicIPAddresses/join/action",
+
+					"Microsoft.Network/routeTables/write",
+					"Microsoft.Network/routeTables/delete",
+					"Microsoft.Network/routeTables/join/action",
+					"Microsoft.Network/routeTables/routes/write",
+					"Microsoft.Network/routeTables/routes/delete",
+
+					"Microsoft.Network/networkSecurityGroups/write",
+					"Microsoft.Network/networkSecurityGroups/delete",
+					"Microsoft.Network/networkSecurityGroups/join/action",
+					"Microsoft.Network/networkSecurityGroups/securityRules/write",
+					"Microsoft.Network/networkSecurityGroups/securityRules/delete",
+
+					"Microsoft.Network/virtualNetworks/subnets/write",
+					"Microsoft.Network/virtualNetworks/subnets/delete",
+					"Microsoft.Network/virtualNetworks/subnets/joinLoadBalancer/action",
+					"Microsoft.Network/virtualNetworks/subnets/join/action",
+
+					"Microsoft.Network/virtualNetworks/write",
+					"Microsoft.Network/virtualNetworks/delete",
+					"Microsoft.Network/virtualNetworks/joinLoadBalancer/action",
+					"Microsoft.Network/virtualNetworks/join/action",
+
+					"Microsoft.Compute/virtualMachines/write",
+					"Microsoft.Compute/virtualMachines/delete",
+					"Microsoft.Compute/virtualMachines/redeploy/action",
+					"Microsoft.Compute/virtualMachines/restart/action",
+					"Microsoft.Compute/virtualMachines/start/action",
+					"Microsoft.Compute/virtualMachines/extensions/write",
+					"Microsoft.Compute/virtualMachines/extensions/delete",
+					"Microsoft.Compute/virtualMachineScaleSets/write",
+					"Microsoft.Compute/virtualMachineScaleSets/delete",
+					"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/write",
+					"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/delete",
+					"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/start/action",
+					"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/redeploy/action",
+					"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/restart/action",
+					"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/extensions/write",
+					"Microsoft.Compute/virtualMachineScaleSets/virtualMachines/extensions/delete",
+					"Microsoft.Compute/virtualMachineScaleSets/delete/action",
+					"Microsoft.Compute/virtualMachineScaleSets/start/action",
+					"Microsoft.Compute/virtualMachineScaleSets/redeploy/action",
+					"Microsoft.Compute/virtualMachineScaleSets/scale/action",
+					"Microsoft.Compute/virtualMachineScaleSets/extensions/write",
+					"Microsoft.Compute/virtualMachineScaleSets/extensions/delete",
```

Comment:
Is these required? Currently ARO do not support VMSS

Reply:
Removed VMSS permissions.